src/PerfView/EventViewer/EventWindow.xaml

```diff
@@ -17,6 +17,7 @@
     </Window.Resources>
     <DockPanel Background="{DynamicResource BackgroundColour}">
         <DockPanel.CommandBindings>
+            <CommandBinding Command="{x:Static src:EventWindow.ToggleMultiLineViewPaneCommand}" Executed="DoToggleMultiLineViewPane" />
```

Comment:
I think we need a way to make sure users know how to toggle this view.  Would it be possible to show some text in the window that says something like "Press F2 to hide." whenever there is not any data shown?